server/public/styles.css

```diff
@@ -83,3 +74,82 @@ textarea {
 i.update-symbol {
   font-size: 1rem;
 }
+
+img {
+  height: 10rem;
+  object-fit: cover;
+}
+
+.category-title {
+  text-align: center;
```

Comment:
If you are using Bootstrap, you should try to find helper classes for each of your CSS properties. Their utilities are bursting at the seams. border, text-alignment, position, flex and much more.


https://getbootstrap.com/docs/4.5/utilities